dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/ResetPasswordTest.php

```diff
@@ -0,0 +1,233 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Customer;
+
+use Exception;
+use Magento\Customer\Api\AccountManagementInterface;
+use Magento\Customer\Model\AccountManagement;
+use Magento\Customer\Model\CustomerRegistry;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+/**
+ * Class reset password for customer account
+ */
+class ResetPasswordTest extends GraphQlAbstract
```

Comment:
We need to add `locked` scenario here as well.

@nrkapoor need to understand expected behaviour here.

Reply:
@lenaorobei - My understanding is that the account is locked for a certain time period (10 minutes by default). During the locked state the reset password functionality must not work.

Reply:
**Done**